common/ch_channel_map/oracular

```diff
@@ -0,0 +1,18 @@
+# NOTE(ganso): This was copy-pasted from noble
```

Comment:
maybe use a symbolic link rather than copying? then we dont have to update multiple files.

Reply:
right but the point is not to have duplicate files so use symlink until they need to diverge